onboarding/forms.py

```diff
@@ -15,7 +15,7 @@
 # new validation logic. The downside is that the old
 # session's onboarding data will disappear, but hopefully
 # we won't have to do this often.
-FIELD_SCHEMA_VERSION = 2
+FIELD_SCHEMA_VERSION = 3
```

Comment:
I really should have done this in #387 but I completely forgot it existed.  I don't think we really need it anymore, since this PR reverts the field schema to what it was before #387, but I figured I'd change it just in case.